web/src/features/map/map-layers/ZonesLayer.tsx

```diff
@@ -39,7 +39,22 @@ export default function ZonesLayer() {
         'fill-color': '#FFFFFF',
         'fill-opacity': ['case', ['boolean', ['feature-state', 'hover'], false], 0.3, 0],
       } as mapboxgl.FillPaint,
+      zonesSelected: {
+        'fill-color': [
+          'coalesce',
+          ['feature-state', 'color'],
+          ['get', 'color'],
+          theme.clickableFill,
+        ],
+        'fill-opacity': [
+          'case',
+          ['boolean', ['feature-state', 'selected'], false],
+          0.8,
```

Comment:
Maybe try a little smaller value here, perhaps 0.7.

Reply:
Yeah I think that's a good balance, will update now